website/src/documentation/transforms/python/element-wise/regex.md

```diff
@@ -19,10 +19,151 @@ limitations under the License.
 -->
 
 # Regex
+
+<script type="text/javascript">
+localStorage.setItem('language', 'language-py')
+</script>
+
 Filters input string elements based on a regex. May also transform them based on the matching groups.
 
 ## Examples
-See [BEAM-7389](https://issues.apache.org/jira/browse/BEAM-7389) for updates. 
 
-## Related transforms 
-* [Map]({{ site.baseurl }}/documentation/transforms/python/elementwise/map) applies a simple 1-to-1 mapping function over each element in the collection
\ No newline at end of file
+In the following examples, we create a pipeline with a `PCollection` of text strings.
+Then, we use the `re` module to search, replace, and split through the text elements using
+[regular expressions](https://docs.python.org/3/library/re.html).
+
+You can use tools to help you create and test your regular expressions such as
+[regex101](https://regex101.com/),
+make sure to specify the Python flavor at the left side bar.
```

Comment:
Split into a new sentence.

"Make sure to specify the Python flavor at the left side bar."

Reply:
Done